packaging/sources/preupgrade-assistant-scripts-rhel5.patch

```diff
@@ -0,0 +1,10 @@
+diff --git a/data/preassessment/scripts.txt b/data/preassessment/scripts.txt
+index 80e2e33..c057834 100644
+--- a/data/preassessment/scripts.txt
++++ b/data/preassessment/scripts.txt
+@@ -1,4 +1,4 @@
+-rpm -qa --qf "%{NAME}\t%{VENDOR}\t%|RSAHEADER?{%{RSAHEADER:pgpsig}}:{(none)}|\n"=rpm_qa.log=RPM_QA=All installed packages=YES=RHEL6_All_installed_packages
++rpm -qa --qf "%{NAME}\t%{VENDOR}\t%|DSAHEADER?{%{DSAHEADER:pgpsig}}:{(none)}|\n"=rpm_qa.log=RPM_QA=All installed packages=YES=RHEL5_All_installed_packages
```

Comment:
Make it more general - to find both DSA and RSA:
rpm -qa --qf "%{NAME}\t%{VENDOR}\t%|DSAHEADER?{%{DSAHEADER:pgpsig}}:{%|RSAHEADER?{%{RSAHEADER:pgpsig}}:{(none)}|}|\n"